backend/tests/unit/dal/Services/ProjectReportServiceTest.cs

```diff
@@ -686,7 +686,7 @@ public void Update_NewToDate()
             // Assert
             Assert.NotNull(result);
             Assert.True(result.Id > 0);
-            result.Should().Equals(reportToUpdate);
+            result.To.Should().Be(reportToUpdate.To.Value, DateTime.UtcNow.Date.ToString());
```

Comment:
Had to replace the equals method in these tests. This would fail if using the "Be" comparing the reportToUpdate, so needed to just compare against the "To" date field itself.